jvb/rootfs/defaults/jvb.conf

```diff
@@ -51,6 +52,16 @@ videobridge {
             port = 9090
         }
     }
+
+    {{ if $ENABLE_OCTO -}}
+    octo {
+        enabled = true
```

Comment:
@saghul I'm not using `enabled = {{{ $ENABLE_OCTO }}` here because in the [reference.conf](https://github.com/jitsi/jitsi-videobridge/blob/3fa0b7c2e3e69b783ad12e89cc94bfd0175f384b/jvb/src/main/resources/reference.conf#L115) its value is `false`  hence no need to use env variable here.